test/websocket/diagnostics-channel.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const { test } = require('tap')
+const dc = require('diagnostics_channel')
+const { WebSocketServer } = require('ws')
+const { WebSocket } = require('../..')
+
+test('undici:websocket:open and undici:websocket:close', (t) => {
```

Comment:
Is it possible to run tap tests sequentially within a file? Otherwise the diagnostics channel were being triggered by both tests, causing them to fail.

Reply:
try `t.jobs = 1` it should force sequential execution

Reply:
The issue was that after each test I wasn't removing the listener 😐.